pages/DSNP/DSNP-Announce.md

```diff
@@ -0,0 +1,62 @@
+# DSNP Announce
+## Specification Status
+
+| Version | Status |
+---------- | ---------
+| 0.1     | Draft |
+
+## Purpose
+1. Describe the method to announce [DSNP Messages](/DSNP/DSNP-Messages) to the blockchain.
+1. Provide Solidity code examples (pending final contract)
+1. Facilitate use of SDK and locating of on-chain data
+
+## Assumptions
+* Chain messages are on Ethereum
+* Message data is posted via [Ethereum log events](https://medium.com/mycrypto/understanding-event-logs-on-the-ethereum-blockchain-f4ae7ba50378)
+* Signature algorithm is [secp256k1](https://en.bitcoin.it/wiki/Secp256k1). This allows the use `ecreover`
+  to get public keys. A public key also need not be included in a log event for ease of validation.
+* content hashes are created via the same [keccak-256 hashing algorithm](https://en.wikipedia.org/wiki/SHA-3) used by Ethereum.
+
+## DSNP Announce
+
+### DSNPMessage Event
+
+The DSNPMmessage event is a standard event topic to be used for all DSNP Messages.
+
+```solidity
+interface DSNPAnnounce {
+    event DSNPMessage(address from, uint16 dsnpType, bytes32 data);
+}
+```
+
+| field | description | type |
+|-------|-------------|------|
+| from | Social Identity From Address | address / uint160 / bytes20 |
+| dsnpType | Enum from [DSNP Message Types](/DSNP/DSNP-Message-Types) | unit8 |
+| data | Matching serialized data from [DSNP Messages](/DSNP/DSNP-Messages) | bytes |
```

Comment:
Naming is hard. Over in the messaging spec this is referred to as `dsnpData`, but that feels redundant in the context of a DSNPMessage event.

Reply:
@rosalinekarr and I chatted about this a little and agreed that naming is hard. I chose dsnp___ because even though it's stuttery, when it's viewed within a documentation context it will never be confusing as to what it is.

Reply:
but similarly to above, I also considered "data" and "messageData"  I just remembered, picked dsnp___ also because it's shorter than message.

Reply:
I wonder how `dsnp_____` holds up against other `dsnp` events we will have, but I think it works for now. We can change it later without it effecting the log event topic (which is signature based, but does not include the parameter names)

Reply:
After thinking it over I think I like the dsnp prefixes (and have perhaps tried to over use them now.)